lib/copyable/declarations/associations.rb

```diff
@@ -11,7 +11,7 @@ def execute(association_list, original_model, new_model, global_override = {}, s
           @skip_associations = skip_associations
           @global_override = global_override
           association_list.each do |assoc_name, advice|
-            association = original_model.class.reflections[assoc_name.to_sym]
+            association = original_model.class.reflections[assoc_name.to_s]
```

Comment:
Is this an AR change, using strings instead of symbols as hash keys?

Reply:
yes :P